### ISSUE TITLE
Can I share this mod with the minecraft forum in China?

### ISSUE THREAD

**Issue by 6Bpencle** [bdew/ae2stuff#182]
The mod is very interesting ,I very like it.I think you can do it better.So I want to share this mod with the minecraft forum in China.This is [website](http://www.mcbbs.net/thread-786713-1-1.html),and let more friends use it.I hope you can agree!
I can abide by the following terms：
1.Clarified origin author name which is YOU(I just a "reposter")
2.Clarified origin URL link could reach AE2 stuff official web/Curse Forge directly
3.Obey EULA with no conditions
4.Never re-compile/re-package/re-distribute this mod
5.A free/AD-free Chinese net-disk download link to boost download speed(No cent could be earn)

**Comment by saymeen94** [comment]
Yeah you can, please check a blog post in mcpedl [website](https://mcpedl.to/)
